clients/spark/core/src/main/scala/io/treeverse/clients/LakeFSInputFormat.scala

```diff
@@ -123,7 +124,9 @@ class EntryRecordReader[Proto <: GeneratedMessage with scalapb.Message[Proto]](
 
   override def getCurrentValue = new WithIdentifier(item.id, item.message, rangeID)
 
-  override def close() = Unit
+  override def close(): Unit = {
+    sstableReader.close()
```

Comment:
I think we _still_ **also** want to delete the file: deleteOnExit says to delete the file when the JVM terminates. For a Spark worker, that can be a loooooong while.

Reply:
added both